packages/actions-global/src/hooks.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  showNotification,
+  showUnexpectedErrorNotification,
+  showApiErrorNotification,
+} from './global';
+import { useDispatch } from 'react-redux';
+
+// Returns a function that dispatches an API error notification.
+// Example:
+//   const showApiErrorNotification = useShowApiErrorNotification();
+//   showApiErrorNotification({ errors });
+export function useShowApiErrorNotification() {
```

Comment:
Nitpick: I would like to have one hook per file. It might sounds not needed or overengineered but it comes at the benefit that I can enter `useShowApiErrorNotification` and get to the file.

Reply:
Sure, I can split them. I'll do that for the global actions as well.